### PR TITLE
Add Databus v6 samples

### PR DESCRIPTION
**Pull request by kentdr**

### CODE REVIEW

File: samples/databus/custom-serializer/Core_9/Receiver/Program.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        var endpointConfiguration = new EndpointConfiguration("Samples.DataBus.Receiver");
+
+        #region ConfigureReceiverCustomDataBusSerializer
+
+        var dataBus = endpointConfiguration.UseDataBus<FileShareDataBus, BsonDataBusSerializer>();
+        dataBus.BasePath(@"..\..\..\..\storage");
+
+        #endregion
+
+        endpointConfiguration.UseSerialization<XmlSerializer>();
```
Comment (andreasohlund) [inline samples/databus/custom-serializer/Core_9/Receiver/Program.cs:20]: is there a reason to use Xml here?

File: samples/databus/custom-serializer/Core_9/Sender/Program.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        var endpointConfiguration = new EndpointConfiguration("Samples.DataBus.Sender");
+
+        #region ConfigureSenderCustomDataBusSerializer
+
+        var dataBus = endpointConfiguration.UseDataBus<FileShareDataBus, BsonDataBusSerializer>();
+        dataBus.BasePath(@"..\..\..\..\storage");
+
+        #endregion
+
+        endpointConfiguration.UseSerialization<XmlSerializer>();
```
Comment (andreasohlund) [inline samples/databus/custom-serializer/Core_9/Sender/Program.cs:22]: Same here

File: samples/databus/databus-custom-serializer-converter/Core_9/Receiver/Program.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.DataBus.Receiver";
+        var endpointConfiguration = new EndpointConfiguration("Samples.DataBus.Receiver");
+        var dataBus = endpointConfiguration.UseDataBus<FileShareDataBus, SystemJsonDataBusSerializer>();
+        dataBus.BasePath(@"..\..\..\..\storage");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```
Comment (andreasohlund) [inline samples/databus/databus-custom-serializer-converter/Core_9/Receiver/Program.cs:15]: Learning persistence shouldn't be needed

File: samples/databus/databus-custom-serializer-converter/Core_9/Sender/Program.cs [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        var endpointConfiguration = new EndpointConfiguration("Samples.DataBus.Sender");
+
+        #region ConfigureDataBus
+
+        var dataBus = endpointConfiguration.UseDataBus<FileShareDataBus, SystemJsonDataBusSerializer>();
+        dataBus.BasePath(@"..\..\..\..\storage");
+
+        #endregion
+
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```
Comment (andreasohlund) [inline samples/databus/databus-custom-serializer-converter/Core_9/Sender/Program.cs:21]: Learning persistence shouldn't be needed

File: samples/databus/databus-custom-serializer-converter/Core_9/Shared/DatabusPropertyConverterFactory.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Shared
+{
+    #region DatabusPropertyConverterFactory
+    public class DatabusPropertyConverterFactory : JsonConverterFactory
+    {
+
```
Comment (andreasohlund) [inline samples/databus/databus-custom-serializer-converter/Core_9/Shared/DatabusPropertyConverterFactory.cs:16]: Nit: remove blankline

File: samples/databus/databus-custom-serializer-converter/Core_9/Shared/DatabusPropertyConverterFactory.cs [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        public override bool CanConvert(Type typeToConvert)
+        {
+            if (!typeToConvert.IsGenericType)
+            {
+                return false;
+            }
+            if (typeToConvert.GetGenericTypeDefinition() != typeof(DataBusProperty<>))
+            {
+                return false;
+
```
Comment (andreasohlund) [inline samples/databus/databus-custom-serializer-converter/Core_9/Shared/DatabusPropertyConverterFactory.cs:26]: Nit: remove blankline

File: samples/databus/file-share-databus/Core_9/Receiver/Program.cs [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.DataBus.Receiver";
+        var endpointConfiguration = new EndpointConfiguration("Samples.DataBus.Receiver");
+        var dataBus = endpointConfiguration.UseDataBus<FileShareDataBus, SystemJsonDataBusSerializer>();
+        dataBus.BasePath(@"..\..\..\..\storage");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```
Comment (andreasohlund) [inline samples/databus/file-share-databus/Core_9/Receiver/Program.cs:13]: Shouldn't be needed?

File: samples/databus/file-share-databus/Core_9/Receiver/Program.cs [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+        endpointConfiguration.UseSerialization<XmlSerializer>();
```
Comment (andreasohlund) [inline samples/databus/file-share-databus/Core_9/Receiver/Program.cs:14]: Can we use json?

**Review by andreasohlund — APPROVED** [review]
Only some nitpicks

**Review by bording — APPROVED** [review]

**Pull request merged**